index.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home / Twitter</title>
+</head>
+<body>
+    <div class="layout">
+        <div class="left">
+            <div> <a class="logo" href="https://www.twitter.com">
```

Comment:
<a> you noramlly want to open in a different page/tab so use target="_blank" inside the tag to do this